### PR TITLE
feat(core): add pause/resume to managed resource menu

### PR DESCRIPTION
**Pull request by anotherchrisberry**
It's kind of weird/rude to only let people pause/resume from a popover IMO

### CODE REVIEW

**Comment by anotherchrisberry** [comment]
Suggestions on what to show when the application is paused? Some options:

1) Screen Shot 2019-12-19 at 5 12 53 PM
(a disabled "Resume Management" option with a help bubble explaining that the app is paused containing a link to resume app management that goes to the config screen)

2) Screen Shot 2019-12-19 at 4 24 18 PM

3) nothing at all, i.e. only show "History" and "Raw Source" options

File: app/scripts/modules/core/src/help/HelpField.tsx [hunk 1]
```diff
@@ -45,7 +45,7 @@ export function HelpField(props: IHelpFieldProps) {
   const contents = <HelpFieldContents content={content} fallback={fallback} id={id} />;
   const popover = (
     <HoverablePopover placement={placement || 'top'} template={contents} onShow={onShow} onHide={onHide}>
-      <a className="clickable help-field"> {label || icon} </a>
```
Comment (erikmunson) [inline app/scripts/modules/core/src/help/HelpField.tsx]: ytho

File: app/scripts/modules/core/src/managed/ManagedResourceDetailsIndicator.tsx [hunk 2]
```diff
@@ -45,6 +53,25 @@ export const ManagedResourceDetailsIndicator = ({ resourceSummary }: IManagedRes
     </>
   );
 
+  // events are getting trapped by React bootstrap menu
+  const allowNavigation = (e: React.MouseEvent) => {
+    const target = e.target as HTMLElement;
+    if (target && target.hasAttribute('href')) {
+      window.location.href = target.getAttribute('href');
+    }
+  };
```
Comment (erikmunson) [inline app/scripts/modules/core/src/managed/ManagedResourceDetailsIndicator.tsx]: oh hey. this is gross. carry on.

**Comment by gcomstock** [comment]
> 👍 👍 Very nice. I don't love the way the multi-layered dropdown + popover looks aesthetically, but I do think it's better in terms of knocking people out of their muscle memory when something is non-normal (vs. surprising them by navigating to the config view when they might not have read the grey text about the app being paused).
> 
> Seems good to merge for now, I will let @gcomstock make any suggestions/tweaks he sees fit given that I can't think of anything more visually appealing that we already do elsewhere.

It's unclear to me what we settled on from this PR. I'll sync up with Chris B!

**Pull request merged**